git-credential.go

```diff
@@ -135,6 +135,14 @@ func filter(ls []string, prefix string) []string {
 	return out
 }
 
+func composePath(c *cli.Context, cred *gitCredentials) string {
+	store := c.String("store") + "/"
+	if store == "/" {
+		store = ""
+	}
+	return store + "git/" + fsutil.CleanFilename(cred.Host) + "/" + fsutil.CleanFilename(cred.Username)
```

Comment:
unrelated to this PR, but I think we need to rethink what cleanups `fsutil.CleanFilename` applies.

Reply:
/sign 
I'll test the windows way in gopass and then collect it in a new pull request agains the open issue.